src/resources/webhooks.ts

```diff
@@ -0,0 +1,138 @@
+// File generated from our OpenAPI spec by Stainless.
+
+import { APIResource } from 'orb-billing/resource';
+import { createHmac } from 'crypto';
+import { getRequiredHeader, HeadersLike } from 'orb-billing/core';
+
+export class Webhooks extends APIResource {
+  /**
+   * Validates that the given payload was sent by Orb and parses the payload.
```

Comment:
I think we should also add this documentation here as well.

```suggestion
   * Validates that the given payload was sent by Orb and parses the payload.
   *
   * An error will be raised if the webhook payload was not sent by Orb.
```